user-guide/Basic_Functionality/Visio/tutorials/Using_Visual_Overview_to_switch_spectrum_presets.md

```diff
@@ -0,0 +1,244 @@
+
+1. Configure the new shape data field with the label `ComponentOptions` and the value `preset=NoInfoPane`, and click *OK*.
+
+1. Make sure no more shapes are selected.
+
+1. Save the Visio file.
+
+1. Check the *Visual* page in DataMiner Cube.
+
+You will see that in DataMiner Cube, the spectrum control no longer includes the info pane.
+
+## Step 7: Create presets for the measurements
+
+In this step, you will create additional presets for each of the center frequencies from which the frequency spectrum should be retrieved.
+
+1. In DataMiner Cube, navigate to the *Spectrum Analyzer* data page of your spectrum simulation element.
+
+1. In the *Manual* tab of the spectrum sidebar, change the settings to the desired values. e.g. for a satellite signal this could be around 12GHz center frequency and 200MHz bandwidth.
+
+   <!-- Add screenshot of configuration -->
+
+1. In the spectrum sidebar, select the *Presets* tab and click the *Manage* button at the bottom.
+
+1. Click the *New* button.
+
+1. Configure the preset:
+
+   - Give the preset the name `Satellite`.
+
+   - Select the *Share preset with other users* checkbox.
+
+   - Click *OK*.
+
+   <!-- Add screenshot of preset configuration before you click OK -->
+
+1. Click *OK* again to close the *Manage presets* pane.
+
+1. Repeat the steps above for a second signal. E.g. For a cable signal, we could use a center frequency of 260 MHz and a Bandwidth of 8 MHz. Give it the name `Cable`.
+
+   <!-- Add screenshot of configuration -->
+
+## Step 8: Use session variables to link the presets to buttons
+
+In this step, you will use session variables and placeholders to link buttons to the presets.
+
+1. In DataMiner Cube, navigate back to the *Visio spectrum presets* view.
+
+1. Go back to the Visio file you were working on earlier.
+
+   If you have already closed the Visio file, re-open it by right-clicking the *Visual* page on the view card and selecting *Edit in Visio*.
+
+1. Resize the shape linked to the spectrum analyzer control to make room for buttons.
+
+   <!-- Add screenshot of Visio page with resized control -->
+
+1. Create a small rectangular shape, and add the text `Switch to Satellite` in the shape.
+
+1. Add a shape data field to this shape, with label `SetVar` and value `varPreset:Satellite (public)`.
+
+   In this value, `varPreset` is the name of the session variable, and `Satellite (public)` refers to the name of the spectrum preset. The (public) suffix is needed to indicate that this is a shared preset.
+
+   <!-- Add screenshot with the new shape selected so that the configured shape data are visible -->
+
+1. Duplicate this shape and change the text and shape data so it refers to the `Cable` preset.
+
+   <!-- Add screenshot with the new shape selected so that the configured shape data are visible -->
+
+1. In the spectrum component shape, replace the fixed preset reference by a session variable placeholder. The new value becomes `preset=[var:varPreset]`
```

Comment:
This step will also need some more detail, as you may be going too fast for some users here. I needs to be clear how they get to the shape data and what exactly they should change.

Reply:
detail and basic explanation of session variables and placeholders added.